web/api/v1/api_test.go

```diff
@@ -3300,10 +3369,35 @@ func BenchmarkRespond(b *testing.B) {
 			},
 		},
 	}
+	request, _ := http.NewRequest("GET", "http://localhost:9090", nil)
+	b.ResetTimer()
+	api := API{}
+	for n := 0; n < b.N; n++ {
+		api.respond(&testResponseWriter, request, response, nil)
+	}
+}
+
+func BenchmarkRespondArrow(b *testing.B) {
+	b.ReportAllocs()
+	points := []promql.Point{}
+	for i := 0; i < 10000; i++ {
+		points = append(points, promql.Point{V: float64(i * 1000000), T: int64(i)})
+	}
+	response := &queryData{
+		ResultType: parser.ValueTypeMatrix,
+		Result: promql.Matrix{
+			promql.Series{
+				Points: points,
+				Metric: nil,
+			},
+		},
+	}
+	request, _ := http.NewRequest("GET", "http://localhost:9090", nil)
+	request.Header.Add("Accept", "application/vnd.apache.arrow.stream")
```

Comment:
Can we add constants for the Accept header value? (and name)

Reply:
I moved the MIME type to a const, but left the header name as a const string as that seems to be what we do elsewhere in this package and Accept and Content-Type are never going to change. Does that seem reasonable?